### PR TITLE
removed payouts dashboard for YPP

### LINKED ISSUE #3706: Hide my payments dashboard for YPP release
ssia

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1203868726963296/1203870592708229) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by attemka**
removed payouts dashboard for YPP
closes #3706

### CODE REVIEW

File: packages/atlas/src/components/_navigation/SidenavStudio/SidenavStudio.tsx [hunk 1]
```diff
@@ -39,12 +39,16 @@ const studioNavbarItems: NavItemType[] = [
...
+  ...(atlasConfig.features.ypp.googleConsoleClientId
+    ? []
+    : [
+        {
+          icon: <SvgSidebarPayments />,
+          name: 'Payments',
+          expandedName: 'My payments',
+          to: absoluteRoutes.studio.payments(),
+        },
+      ]),
```
Comment (drillprop) [inline packages/atlas/src/components/_navigation/SidenavStudio/SidenavStudio.tsx]: I don't think we should just hide the payments from the navigation. On gleev we still have payments in navigation, although it's not working - we have a placeholder screen there.
image

Removing the payments nav item together with the placeholder screen could suggest to the users that we abandoned the payments feature when we release YPP.

**Review by drillprop — CHANGES_REQUESTED** [review]
Unfortunately, this is not enough. If I switch the environment to atlas-dev, then I connect to the wallet and go to the studio, the app will break.
image

Remember that we want to use the dev environment for testing.

Besides that, we should also change the targetDevEnv inside `packages/atlas/src/providers/environment/store.ts`

**Comment by dmtrjsg** [comment]
+ there's one event that is taken from Ephesus and it needs to be removed from Graph QL

File: packages/atlas/src/components/_navigation/SidenavStudio/SidenavStudio.tsx (hunk 1 above)
Comment (drillprop) [reply packages/atlas/src/components/_navigation/SidenavStudio/SidenavStudio.tsx]: You brought back the placeholder view but didn't bring back the nav item in the sidebar.

**Review by drillprop — CHANGES_REQUESTED** [review]
Also, let's update the targetDevEnv inside `packages/atlas/src/providers/environment/store.ts` to 'development'. 

This will set the `development` environment as default(currently it's `next`)

**Review by drillprop — APPROVED** [review]
LGTM

**Pull request merged**